test/fixtures/vies_responses.ex

```diff
@@ -25,47 +25,64 @@ defmodule Fixtures.VIESResponses do
 
   def valid_vat_response do
     """
-    <soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
-      <soap:Body>
-        <checkVatResponse xmlns="urn:ec.europa.eu:taxud:vies:services:checkVat:types">
-          <countryCode>GB</countryCode>
-          <vatNumber>333289454</vatNumber>
-          <requestDate>2016-01-16+01:00</requestDate>
-          <valid>true</valid>
-          <name>BRITISH BROADCASTING CORPORATION</name>
-          <address>BC0 B1 D1 BROADCAST CENTRE\nWHITE CITY PLACE\n201 WOOD LANE\nLONDON\n\nW12 7TP</address>
-        </checkVatResponse>
-      </soap:Body>
-    </soap:Envelope>
+    <env:Envelope xmlns:env="http://schemas.xmlsoap.org/soap/envelope/">
+      <env:Header/>
+      <env:Body>
+        <ns2:checkVatResponse xmlns:ns2="urn:ec.europa.eu:taxud:vies:services:checkVat:types">
+          <ns2:countryCode>GB</ns2:countryCode>
+          <ns2:vatNumber>333289454</ns2:vatNumber>
+          <ns2:requestDate>2016-01-16+01:00</ns2:requestDate>
+          <ns2:valid>true</ns2:valid>
+          <ns2:name>BRITISH BROADCASTING CORPORATION</ns2:name>
+          <ns2:address>BC0 B1 D1 BROADCAST CENTRE\nWHITE CITY PLACE\n201 WOOD LANE\nLONDON\n\nW12 7TP</ns2:address>
+        </ns2:checkVatResponse>
+      </env:Body>
+    </env:Envelope>
     """
   end
 
   def invalid_vat_response do
     """
-    <soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
-      <soap:Body>
-        <checkVatResponse xmlns="urn:ec.europa.eu:taxud:vies:services:checkVat:types">
-          <countryCode>GB</countryCode>
-          <vatNumber>123123123</vatNumber>
-          <requestDate>2016-01-16+01:00</requestDate>
-          <valid>false</valid>
-          <name>---</name>
-          <address>---</address>
-        </checkVatResponse>
-      </soap:Body>
-    </soap:Envelope>
+    <env:Envelope xmlns:env=\"http://schemas.xmlsoap.org/soap/envelope/\">
+      <env:Header/>
+      <env:Body>
+        <ns2:checkVatResponse xmlns:ns2=\"urn:ec.europa.eu:taxud:vies:services:checkVat:types\">
+          <ns2:countryCode>GB</ns2:countryCode>
+          <ns2:vatNumber></ns2:vatNumber>
```

Comment:
Testing revealed this now returns an empty value instead of the original `vatNumber` for an invalid VAT response.